dll/win32/devmgr/devmgmt/ClassNode.cpp

```diff
@@ -42,14 +42,12 @@ CClassNode::SetupNode()
                                      0);
     if (hKey != INVALID_HANDLE_VALUE)
     {
-        Type = REG_SZ;
-
         // Lookup the class description (win7+)
         Size = sizeof(m_DisplayName);
         Success = RegQueryValueExW(hKey,
                                    L"ClassDesc",
                                    NULL,
-                                   &Type,
+                                   NULL,
                                    (LPBYTE)m_DisplayName,
                                    &Size);
         if (Success == ERROR_SUCCESS)
```

Comment:
It looks like the original intention was to verify the type, so I think the fix should be to do that: add `&& Type == REG_SZ` here (and set an appropriate error if it's not).

Reply:
Now checking out value.